src/main.rs

```diff
@@ -65,3 +82,22 @@ fn handle_error(e: cli::error::Error) {
 
     exit(1);
 }
+
+// Prints a warning/info message if this version of amethyst_cli is out of date
+fn check_version() -> cli::error::Result<()>{
+    use cli::get_latest_version;
+    use ansi_term::Color;
+
+    let local_version = semver::Version::parse(env!("CARGO_PKG_VERSION"))?;
+    let remote_version_str = get_latest_version()?;
```

Comment:
I'm a bit concerned about the program taking seconds to create a project if the internet connection is bad.

Reply:
I added the skeleton of an "update" subcommand that, for now, checks our version of amethyst-tools to see if it can be updated. It doesn't currently do any updating. I removed the mandatory check and it now gets run if you type "amethyst update"